packages/core/launch.ts

```diff
@@ -791,7 +791,7 @@ export async function generateArguments(options: LaunchOption) {
   }
 
   if (options.prependCommand && options.prependCommand.trim().length > 0) {
-    cmd.unshift(options.prependCommand.trim())
+    cmd.unshift(...options.prependCommand.trim().split(' '))
```

Comment:
would u mind to do a full fix?

like making prependCommand a `string | string[]` and allow string array here?

so if the prepend filename contains space wont be buggy